### PR TITLE
Fix DELETE methods status code

### PR DESCRIPTION
**Pull request by JakubFrejlach**
This PR fixes discrepancy between OSIDB API schema and actual returned status code from DELETE methods.

### CODE REVIEW

**Review by osoukup — APPROVED** [review]
LGTM

**Comment by JakubFrejlach** [comment]
I see there is a backward compatibility check failed, howerver this is actually a false positive, as we simple advertised something which was not true (eg. schema advertising return code 204, but in fact return code is 200), should we bypass that in this case?

**Comment by osoukup** [comment]
> @osoukup I see there is a backward compatibility check failed, howerver this is actually a false positive, as we simple advertised something which was not true (eg. schema advertising return code 204, but in fact return code is 200), should we bypass that in this case?

Makes sense to me. @Elkasitu WDYT?

File: osidb/api_views.py [hunk 1]
```diff
@@ -410,6 +410,11 @@ def create(self, request, *args, **kwargs):
 
 
 class SubFlawViewDestroyMixin:
+    @extend_schema(
+        responses={
+            200: {},
```
Comment (Elkasitu) [inline osidb/api_views.py:415]: It can also return 400

Comment (JakubFrejlach) [reply osidb/api_views.py:415]: Well in that case we are missing the 400 and all possible error status codes in every endpoint for every method :laughing: There is a not a single response example for 4xx status code anywhere in the OSIDB currently

Comment (Elkasitu) [reply osidb/api_views.py:415]: Fair enough

**Review by Elkasitu — APPROVED** [review]

**Comment by Elkasitu** [comment]
@JakubFrejlach the check is not marked as required so it should not block merge, are you sure it's not something else that's causing the failure? See #262 which was successfully merged in spite of the failing check

**Comment by JakubFrejlach** [comment]
> @JakubFrejlach the check is not marked as required so it should not block merge, are you sure it's not something else that's causing the failure? See #262 which was successfully merged in spite of the failing check

Ahhh, thank you! It fails do to a schema check not schema backwards compatibility check. I mixed those two together.

**Pull request merged**